### PR TITLE
Fixed Radiobutton Width

### PR DESCRIPTION
**Pull request by krishane2018**
…function. Used constant to define width of radiobuttons of first window.

__What issue are you solving? (Give #issuenumber)__

Change size of radio buttons #62

__How do you solve it? (please write this so a human can understand it - bulletpoints are okay)__

- Made a constant BTTN_WIDTH and defined its value as 18 in the input_buttons function

- Using the width configuration of the radiobuttons, I set both of the width values to BTTN_WIDTH

__If there is something specific in your solution, that you would like the reviewer to give feedback on, tell us here:__

_If this is your first pull request to the prog-o-meter project, we would really appreciate it if you would fill out this Google form.
This lets us know more about our contributors, and what we can do to make it easier and nicer to contribute to the prog-o-meter project.
Please submit your answer [here](https://docs.google.com/forms/d/e/1FAIpQLSdQqs9shjcyJTTqbkKjGv85Irxxaljvw0H357zuyrRwXq9_nA/viewform?usp=sf_link) and put an 'x' in the box below._

- [x] It is my first pull request to prog-o-meter, and I have answered the survey

### CODE REVIEW

File: prog-o-meter.py [hunk 1]
```diff
@@ -195,8 +195,9 @@ def input_buttons(self):
         Displays a set of two radio buttons, for the user to indicate whether they are a new
         or returning user. Window closes when user clicks one of the buttons.
         """
-        Tk.Radiobutton(self.root, text = "I already have a meter", variable = self.choice, value = 1, command = self.close_window, indicatoron = 0).pack(pady = 5)
-        Tk.Radiobutton(self.root, text = "I don't have a meter yet", variable = self.choice, value = 2, command = self.close_window, indicatoron = 0).pack(pady = 5)
+        BTTN_WIDTH = 18
```
Comment (lineaba) [inline prog-o-meter.py]: Just a small detail. For documentation purposes, please add a line after 196 that says

Attributes:
   BTTN_WIDTH: (explanation of the attribute here)

You can see an example of this in the docstring (comment) of the function canvas_layout(self) on line 180

Comment (krishane2018) [reply prog-o-meter.py]: Alright I understand, I will add the documentation.

Comment (krishane2018) [reply prog-o-meter.py]: The documentation has been added.

**Comment by lineaba** [comment]
Thanks for adding the comments. I am merging your contribution! :)

**Pull request merged**